hasher-matcher-actioner/hmalib/lambdas/fetcher.py

```diff
@@ -255,18 +255,19 @@ def load_state(self, allow_cached=True):
                 csv.field_size_limit(65535)  # dodge field size problems
                 for row in csv.reader(txt_content):
                     items.append(
-                        CliIndicatorSerialization(
-                            "HASH_PDQ",
+                        HMASerialization(
                             row[0],
```

Comment:
I missed this in #553 but having the constructor args for `indicator_type` and `indicator` be swapped between `HMASerialization` and `CliIndicatorSerialization` messed with my head. Maybe worth a comment.

Reply:
Ah. Didnt think of that either. A comment would be useful but don't think this is the right place. Probably in the `HMASerialization` class